CONTRIBUTING.md

```diff
@@ -70,28 +76,13 @@ Images are added to the markdown by giving the full file-path, e.g.
 
 ### Acknowledgements
 
-Every package release should include an acknowledgements section individually thanks every major contributor, and collectively thanks all GitHub contributors. You can use the following block of code to get all contributors:
+Every package release should include an acknowledgements section individually thanks every major contributor, and collectively thanks all GitHub contributors. You can use the following block of code to get all contributors to a package since its last release, and paste it into your post:
 
 ```R
-gh_users <- function(owner, repo, since) {
-  x <- gh::gh(
-    "/repos/:owner/:repo/issues", 
-    owner = owner, 
-    repo = repo, 
-    since = since, 
-    state = "all", 
-    .limit = Inf
-  )
-  sort(unique(purrr::map_chr(x, c("user", "login"))))
-}
-
-users <- gh_users("r-dbi", "bigrquery", "2017-06-26")
-length(users)
-ack <- glue::collapse(glue::glue("[\\@{users}](https://github.com/{users})"), ", ", last = ", and ")
-clipr::write_clip(ack)
+use_tidy_thanks("r-lib/usethis", from = "2018-05-01")
 ```
 
-(Make sure to update owner, repo, and since)
+(Make sure to update owner/repo, and from)
```

Comment:
I think you can delete this sentence now

Reply:
Proposal:

> You can use `usethis::use_tidy_thanks()` to get all contributors to a package in a time interval and paste this into your post. Examples:

``` r
use_tidy_thanks("OWNER/REPO") ## default: interval = since the last release
use_tidy_thanks("OWNER/REPO", from = "2018-05-01")
use_tidy_thanks("OWNER/REPO", from = "v1.3.0")
```